test/verify/check-networkmanager-wireguard

```diff
@@ -43,15 +43,28 @@ class TestWireGuard(packagelib.PackageCase, netlib.NetworkCase):
         m2_port = 51820
         m2_ip4 = "10.0.0.2"
         m2_ip6 = "2001::2"
+        b2 = self.new_browser(m2)
+        m2.start_cockpit()
+        b2.login_and_go("/network")
         if not m2.ostree_image:
-            m2.execute(f"firewall-cmd --add-port={m2_port}/udp")
-        m2.execute("wg genkey > private")
-        m2_pubkey = m2.execute("wg pubkey < private").strip()
-        m2.execute("ip link add dev wg0 type wireguard")
-        m2.execute(f"ip addr add {m2_ip4}/24 dev wg0")
-        m2.execute("wg set wg0 private-key ./private")
-        m2.execute(f"wg set wg0 listen-port {m2_port}")
-        m2.execute("ip link set wg0 up")
+            b2.go("/network/firewall")
+            b2.enter_page("/network/firewall")
+            b2.click(".zone-section:contains('Public zone') button:contains('Add services')")
+            b2.wait_visible("#add-services-dialog")
+            b2.set_input_text("#filter-services-input", "wireguard")
+            b2.wait_visible("ul li label:contains('wireguard')")
+            b2.click("ul li label:contains('wireguard')")
+            b2.click("#add-services-dialog button:contains('Add services')")
+            b2.wait_visible("#zones-listing tr:contains('wireguard')")
+            b2.go("/network")
+            b2.enter_page("/network")
```

Comment:
FTR, *this* part is completely fine to keep as `firewall-cmd`, as it's not related to the actual wireguard setup. In fact, it's preferable to not  mix tests of different pages. But it's okay, just a suggestion for simplification.